otter/integration/tests/test_convergence.py

```diff
@@ -67,6 +69,135 @@ def _check_fds(_):
             return deferLater(reactor, 0, _check_fds, None)
         return self.pool.closeCachedConnections().addBoth(_check_fds)
 
+    def test_scale_over_group_max_after_metadata_removal_reduced_grp_max(self):
+        """
+        Attempt to scale over the group max, but only after metadata removal on
+        some random sampling of servers.  Note that this version exercises the
+        path that group max is less than CLB max.
+        """
+
+        rcs = TestResources()
+
+        self.untouchable_scaling_group = ScalingGroup(
```

Comment:
The assertions will verify that what servers existed prior to this test remain there after the completion of the test.  I guess @kivattik had the idea of testing whether or not Convergence would "overflow" excess servers to another scaling group?